### ISSUE TITLE
Database tables not recreated when re-installing plugin

### ISSUE THREAD

**Issue by chopsuei3** [ammmze/DeployMint#16]
I am trying to reinstall DepoyMint from scratch on my multisite instance.
To remove -
1. I deactivated DeployMint plugin on Network Admin dashboard
2. Removed DeployMint directory from plugin folder
3. Deleted all deploymint tables (all tables starting with dep, including dep_options, dep_blogs, and dep_members.

Then I attempted to reinstall the plugin using the installation instructions - 
1. Put DeployMint directory in plugin directory
2. On Network Admin dashboard, activate DeployMint plugin

The plugin gets activated correctly, but when I try to set any of the settings in Options, the fields aren't saved.
I found that the tables (dep_*) aren't recreated when I activate the plugin.

I traced a section of code that looks for 'deploymint_db_version' in the wp_options table, thinking if this value is missing, the dep_\* tables would be created, but that didn't seem to work.

To summarize, how do I force DeployMint to create the database tables if they are not created?

Thanks again bcash!
